### ISSUE TITLE
Textures are extremely high-res and wasteful

### ISSUE THREAD

**Issue by JonnyOThan** [linuxgurugamer/PrakasaAeroworks#4]
There are lots of 1k and 2k textures that really don't need to be.  This is the Prakasa directory:
![image](https://github.com/linuxgurugamer/PrakasaAeroworks/assets/9804078/f5dec9fe-739f-4664-957e-7dc9844bde97)

**Comment by linuxgurugamer** [comment]
Are you going to be fixing this in your PR?

**Comment by JonnyOThan** [comment]
No, I'm not really any good at textures.  But wanted to flag this for the future (tm)
